docs/FRSKY_R9.md

```diff
@@ -1,8 +1,8 @@
-# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9MX #
+# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9 MX #
 
 ([back to main page](../README.md))
 
-The Frsky R9M, R9M Lite Pro transmitter modules and R9MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
+The Frsky R9M, R9M Lite Pro transmitter modules and R9 MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
```

Comment:
R9 MM
shouldn't we maybe list the R9 Mini here too?

Reply:
Added